ColorPicker/ColorPicker.js

```diff
@@ -208,26 +208,26 @@ const ColorPickerBase = kind({
 						>
 							{children}
 						</Group>
-						<Button icon="ellipsis" size="small" onTap={onToggleExtended} className={css.swatch} />
+						<Button aria-label={$L('More')} icon="ellipsis" size="small" onTap={onToggleExtended} className={css.swatch} />
 						<div className={css.sliders}>
 							<Row align="center">
 								<Cell>
 									<label>{$L('Hue')}</label>
-									<Slider value={sliderValues.hsl[0]} min={0} max={360} onChange={onHueChanged} />
+									<Slider aria-label={$L('Hue')} value={sliderValues.hsl[0]} min={0} max={360} onChange={onHueChanged} />
```

Comment:
For now, the screen reader reads value + 'Hue'. 'Hue' + value would be better.
We could do it with `<Cell aria-label={$L('Hue')} role="region">` at 214 line.